.github/workflows/ci.yml

```diff
@@ -86,32 +88,20 @@ jobs:
           rustup component add rustfmt
 
       - name: Install Deno
-        if: |
-          !startsWith(matrix.os, 'windows')
-        run: |-
+        if: matrix.kind == 'lint'
+        run: |
           curl -fsSL https://deno.land/x/install/install.sh | sh -s v1.7.2
           echo "$HOME/.deno/bin" >> $GITHUB_PATH
 
-      - name: Error on Warning
-        run: echo "RUSTFLAGS=-D warnings" >> $GITHUB_ENV
```

Comment:
Removing this disables error on warning.

This could be cleaner if the following was put above
```
env:
  RUSTFLAGS: -D warnings
```